src/Layout/redist/MSBuildImports/Current/Microsoft.Common.CrossTargeting.targets/ImportAfter/Microsoft.TestPlatform.CrossTargeting.targets

```diff
@@ -93,14 +93,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
   ==================================================================================
-                                   _MTPBuild
+                                   _MTPTest
 
    Cross-targeting version of MTP.
 
   =================================================================================
   -->
-
-  <Target Name="_MTPBuild" DependsOnTargets="_ComputeTargetFrameworkItems">
+  <!-- This target is called by dotnet test for MTP for projects that are multi-targeted (using multiple target frameworks) -->
+  <Target Name="_MTPTest" Returns="$(_TestTfmsInParallel)">
+    <PropertyGroup>
+      <_TestTfmsInParallel>$([MSBuild]::ValueOrDefault('$(TestTfmsInParallel)', '$(BuildInParallel)'))</_TestTfmsInParallel>
+      <InnerTargets Condition="'$(MTPNoBuild)'!='true'">Build;_MTPTest</InnerTargets>
+      <InnerTargets Condition="'$(MTPNoBuild)'=='true'">_MTPTest</InnerTargets>
```

Comment:
Note: this doesn't really work because `CallTarget` below is executed in a different scope and can't see the values set here.